Makefile.prow

```diff
@@ -28,15 +30,22 @@ XARGS = xargs -0 ${XARGS_FLAGS}
 CLEANXARGS = xargs ${XARGS_FLAGS}
 
 IMG ?= $(shell cat COMPONENT_NAME 2> /dev/null)
+REGISTRY = quay.io/open-cluster-management
+VERSION ?= $(shell cat COMPONENT_VERSION 2> /dev/null)
+IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG):$(VERSION)
 export GOPACKAGES   = $(shell go list ./... | grep -v /manager | grep -v /bindata  | grep -v /vendor | grep -v /internal | grep -v /build | grep -v /test | grep -v /e2e )
 
 .PHONY: build
 
--include /opt/build-harness/Makefile.prow
-
 build:
 	@common/scripts/gobuild.sh build/_output/bin/$(IMG) ./cmd/manager
 
+.PHONY: build-images
```

Comment:
why don't we include the `Makefile` then just use the `build-images` and `e2e` directly?

something like:
https://github.com/open-cluster-management/applifecycle-backend-e2e/blob/76827d8db5b1be1464410120fc4e30062820e799/Makefile.prow#L16

https://github.com/open-cluster-management/applifecycle-backend-e2e/blob/76827d8db5b1be1464410120fc4e30062820e799/Makefile.prow#L38-L39

Reply:
It's for future proof. `kind` prow support is coming soon according to Ray so I want to isolate the 2 Makefiles.

The idea behind the 2 Makefiles is the .prow one is dedicated for prow so we don't expect community/external contributors to have to make changes to it. While the original Makefile is more open for change.